docusaurus/docs/learn/quickstarts/network/miniziti.bash

```diff
@@ -89,24 +95,35 @@ detectOs(){
     fi
 }
 
-getClientOttPath(){
-    local ID_PATH="/tmp/${MINIKUBE_PROFILE}-client.jwt"
-    if [[ $# -eq 1 ]]; then
-        case $1 in
-            macOS|Linux)     echo "$ID_PATH"
+makeMinizitiXDGStateDir() {
```

Comment:
Slight preference for sticking to dromedary case instead of deviating to a hybrid with some uppercase.

```suggestion
makeMinizitiXdgStateDir() {
```

Reply:
I've simply removed `XDG` from the name because I find that variant repulsive. There is actually no consensus on that subject anywhere I'm aware of. Some projects encode this in style guides. In the future, changes like these should be the cause of churn only if we can point to a specific guideline infringement. Even better, a linter in a pipeline should enforce such conventions.